src/version_handler.rs

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Keylime Authors
+
+use crate::common::API_VERSION;
+use actix_web::{web, HttpRequest, HttpResponse, Responder};
+use log::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+struct KeylimeVersion {
+    supported_version: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct JsonVersionWrapper {
+    code: u32,
+    status: String,
+    results: KeylimeVersion,
+}
+
+impl JsonVersionWrapper {
+    fn new(results: KeylimeVersion) -> Self {
+        JsonVersionWrapper {
+            code: 200,
+            status: String::from("Success"),
+            results,
+        }
+    }
+}
```

Comment:
I would say that, depending on how strict we want to be, at least the structure wrapping the HTTP status could be shared by all responders. The downside I see is losing the strong type checking on the results content as we would replace the `results` type to accept any JSON content instead of a specific type. The same with the relationship between API and response type.

Should I include the change in this PR?

Reply:
Oh, yes, thanks for suggesting that! I'm new to rust, so I'm not used to think in the "rust way". I'll open a new PR to make that change.